guides/common/modules/proc_browsing-repository-content-using-an-organization-debug-certificate.adoc

```diff
@@ -4,7 +4,7 @@
 You can view an organization's repository content using a web browser or using the API if you have a debug certificate for that organization.
 
 .Prerequisites
-. Create and download an organization certificate as described in xref:Creating_an_Organization_Debug_Certificate_{context}[].
+. Create and download an organization certificate as described in {ManagingOrganizationsLocationsDocURL}Creating_an_Organization_Debug_Certificate_managing-organizations-locations[].
```

Comment:
This is what renders for the Content Management Guide in Satellite build:

Create and download an organization certificate as described in https://access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html-single/content_management_guide/index#Creating_an_Organization_Debug_Certificate_managing-organizations-locations

Reply:
This is broken due to the anchor that has been changed, not due to the attribute. Link that works:

```
https://access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html-single/content_management_guide/index#Creating_an_Organization_Debug_Certificate_managing-organizations-locations
https://access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html-single/content_management_guide/index#Managing_Organizations-Creating_an_Organization_Debug_Certificate
```

This brings me to the question: How come our `master` branch still has 6.10 as Satellite URL? See https://github.com/theforeman/foreman-documentation/blame/master/guides/common/attributes.adoc#L17 This cannot work, or am I misunderstanding?

Reply:
That's the current release. If we change to 7, which will be the next release, we break every link and fail every build? 
>This is broken due to the anchor that has been changed
I don't follow. I get that it works without the anchor, but that was my whole point: it isn't working here and now.